src/languages/bigquery.formatter.ts

```diff
@@ -838,10 +836,36 @@ export default class BigQueryFormatter extends Formatter {
         ...Object.values(reservedFunctions).reduce((acc, arr) => [...acc, ...arr], []),
         ...Object.values(reservedKeywords).reduce((acc, arr) => [...acc, ...arr], []),
       ]),
-      stringTypes: BigQueryFormatter.stringTypes,
-      indexedPlaceholderTypes: ['?'],
+      stringTypes: [
+        // The triple-quoted strings are listed first, so they get matched first.
+        // Otherwise the first two quotes of """ will get matched as an empty "" string.
+        '""".."""',
```

Comment:
can we instead pass this as something like
`{quote: '""".."""', prefixes: ['R', 'B', 'BR', 'RB']}` for each quote pattern and have the regexFactory generate all combinations + an implied case without prefixes

this is assuming there are cases in other languages where not all prefixes are exclusive

Reply:
I thought of doing this, but as only BigQuery needs this ability to combine multiple prefixes, it felt simpler to just list all the combinations.

Reply:
oh yes I meant as in, list all combinations per supported quote type
so this list would look like
```
{quote: '""".."""', prefix: ['R', 'B', 'RB', 'BR'] },
{quote: "'''..'''", prefix: ['R', 'B', 'RB', 'BR'] },
{quote: '".."', prefix: ['R', 'B', 'RB', 'BR'] },
{quote: "'.."', prefix: ['R', 'B', 'RB', 'BR'] },
```
not as in give it 'R', 'B' and have it generate 'RB' and 'BR'

Reply:
Yeah, sure, this can be easily done. I'll just need to add a function to expand:
```
{quote: '""".."""', prefix: ['R', 'B', 'RB', 'BR'] },
```

into:

```
{ quote: '""".."""' },
{ prefix: 'R', quote: '""".."""' },
{ prefix: 'B', quote: '""".."""' },
{ prefix: 'RB', quote: '""".."""' },
{ prefix: 'BR', quote: '""".."""' },
```